libmamba/CMakeLists.txt

```diff
@@ -448,10 +448,9 @@ macro(libmamba_create_target target_name linkage deps_linkage output_name)
             ${LIBSOLV_LIBRARIES}
             ${LIBSOLVEXT_LIBRARIES}
             ${LibArchive_LIBRARIES}
-            zstd::libzstd_shared
+            $<IF:$<TARGET_EXISTS:zstd::libzstd_shared>,zstd::libzstd_shared,zstd::libzstd_static>
```

Comment:
MIxing static and shared libraries may lead to undefined behavior because of possibly different runtimes (especially on Windows), in which case the dynamic library could be missing?

Reply:
```zstd``` is built statically on risc-v platforms because ```VCPKG_LIBRARY_LINKAGE``` is ```static``` in triplets ```riscv64-linux```. I will try to build a statically-linked ```mamba``` and roll back this change.